.gitignore

```diff
@@ -1,4 +1,6 @@
+.idea/
+vendor/
+build/
```

Comment:
Wat doet `build/`?

Reply:
Hier niks. Staat wel in meer bundles, zal wel via een copy/paste van een project-variant zijn gekomen.

Reply:
:bulb: Die directory wordt aangemaakt voor coverage reports kom ik zojuist achter